packages/react/src/components/toggle-switch/toggle-switch.tsx

```diff
@@ -65,16 +67,14 @@ const StyledButton = styled.button<StyledButtonProps>`
     }
 
     &:disabled {
-        background: ${({ theme }) => theme.greys.grey};
-        border-color: ${({ theme }) => theme.greys.grey};
+        background: ${({ theme }) => theme.component['toggle-switch-container-disabled-background-color']};
+        border-color: ${({ theme }) => theme.component['toggle-switch-container-disabled-border-color']};
 
         &[aria-checked='true'] {
-            background: ${({ theme }) => theme.notifications['success-1.3']};
-            border-color: ${({ theme }) => theme.notifications['success-1.3']};
+            background: ${({ theme }) => theme.component['toggle-switch-container-disabled-toggled-background-color']};
+            border-color: ${({ theme }) => theme.component['toggle-switch-container-disabled-toggled-border-color']};
```

Comment:
Je suis pas certain lequel des deux devrait être avant 🤔 
```suggestion
            background: ${({ theme }) => theme.component['toggle-switch-toggled-disabled-background-color']};
            border-color: ${({ theme }) => theme.component['toggle-switch-toggled-disabled-border-color']};
```

Reply:
Difficile à dire, mais j'aurais tendance à dire disabled en premier, puisque le state disabled est dans la plupart des cas prioritaire sur le styling... Mais bon on peut philosopher là dessus! La discussion est ouverte!